spark/hbase-spark-it/pom.xml

```diff
@@ -103,7 +103,8 @@
         <artifactId>maven-failsafe-plugin</artifactId>
         <configuration>
           <skip>false</skip>
-          <forkMode>always</forkMode>
+          <forkCount>1</forkCount>
+          <reuseForks>true</reuseForks>
```

Comment:
The [documentation](https://maven.apache.org/surefire/maven-failsafe-plugin/examples/fork-options-and-parallel-execution.html) at _Migrating the Deprecated forkMode Parameter to forkCount and reuseForks_ says forkMode=always is equivalent with forkCount=1 (default) and reuseForks=false. However, not reusing the processes would add an overhead so I think it is a reasonable modification.

Reply:
@petersomogyi in terms of compiling hbase-spark-it individually, I ran mvn clean install on the module first and then I'm able to run mvn clean package on it, both with and without my patch.